eng/versioning/version_client.txt

```diff
@@ -99,30 +99,30 @@ com.microsoft.azure:azure-spring-boot-starter;2.3.4;2.3.5-beta.1
 com.microsoft.azure:azure-spring-boot;2.3.4;2.3.5-beta.1
 com.microsoft.azure:spring-cloud-azure-appconfiguration-config-web;1.2.8-beta.1;1.2.8-beta.1
 com.microsoft.azure:spring-cloud-azure-appconfiguration-config;1.2.8-beta.1;1.2.8-beta.1
-com.microsoft.azure:spring-cloud-azure-context;1.2.8-beta.1;1.2.8-beta.1
```

Comment:
Let's move the app configuration related ones together and sort lines for the rest of the spring cloud modules.